### ISSUE TITLE
sokol_app.h: Resizing window with GLCORE33 on Windows

### ISSUE THREAD

**Issue by dbloodman** [floooh/sokol#414]
First, thank you for developing sokol libraries.

When resizing the window, context does not scale with window. It is scaled when resizing is finished.

This case works on D3D11.

Example:

![Sokol Apps Opengl](https://user-images.githubusercontent.com/9919036/97713232-9c232200-1ad0-11eb-870d-75ff70e250d8.png)

**Comment by floooh** [comment]
Yes, the underlying behaviour is the same as in the D3D11/DXGI backend code (the framebuffer is only resized at the end of the window resizing operation), however DXGI allows to blit the framebuffer with scaling which doesn't look quite as ugly. I haven't found a similar flag for WGL yet (but I haven't looked too hard either).

The "obvious" solution to resize the framebuffer during the window resize unfortunately leads to a video memory spike of up to 500 MBytes which persists for 1 or 2 seconds. I haven't found a solution for yet.

See here:

https://github.com/floooh/sokol/blob/612755ec6665ea71c84aa9ca8a6d2b53f9730293/sokol_app.h#L5747-L5756

Removing the comments there looks "proper" but introduces this memory spike.

**Comment by kilgariff** [comment]
Just FYI, I don't see the GPU memory spike in the task manager while resizing my app after uncommenting the above code. Test app is based on the ImGui sample from [these Zig sokol samples](https://github.com/jeffkdev/sokol-zig-examples). This is the same behaviour regardless of whether I define `SOKOL_D3D11` or `SOKOL_GLCORE33`.

Windows 10 Pro build 21390
NVIDIA GeForce RTX 2080 Super
Driver version 466.47
"Hardware Accelerated GPU Scheduling" feature is enabled

Although the window does resize dynamically, the contents of the window judder as though they're one frame behind.

**Comment by floooh** [comment]
You won't see the spike in the task manager, because it's only short-lived and I'm also not sure if the task manager actually shows memory used by GPU resources). The realtime memory profiler in Visual Studio shows the spike though.

The "one frame behind" thing is because of some latency sources in the input- and render-pipeline. I think some of those can be slightly improved (such mouse event latency), but over others the application doesn't have much control unfortunately.
